engineering-report/document.adoc

```diff
@@ -0,0 +1,62 @@
+= Engineering report for OGC (add title text)
+:comment: ### Document type; mandatory. Visit: https://www.metanorma.com/author/ogc/authoring/ for permitted types
+:doctype: engineering-report
+:comment: ### Language of the document; mandatory. Specified in two-letter code: "en" for English, "fr" for French
+:language: en
+:comment: ### Document status/stage; mandatory. Visit: https://www.metanorma.com/author/ogc/authoring/ for permitted types
+:status: published
+:comment: ### Relevant committee; mandatory. The permitted types are: technical, planning, and strategic-member-advisory
+:committee: technical
+:comment: ### Internal reference number; mandatory
+:docnumber: 18-046
+:comment: ### Date on which the standard was updated; mandatory
+:received-date: 2018-12-20
+:comment: ### Date on which the standard was approved by the issuing authority; mandatory
+:issued-date: 2018-07-10
+:comment: ### Date on which the standard was published; mandatory
+:published-date: 2018-06-12
+:comment: ### External link referencing the document; optional. If not provided, a default value is created following this structure: "http://www.opengis.net/doc/{abbrevation of doctype}/{abbrev}/{version}"
+:external-id: http://www.opengis.net/doc/PER/eoep-Hack2018
+:comment: ### Author one
+:fullname: John Doe
+:comment: ### Author two
+:fullname_2: Jane Doe
+:comment: ### Role of the authors; mandatory
```

Comment:
Not mandatory; supplied only if the author is an editor. To make it clear that it applies to an individual author, make this :role_2:

Reply:
Done, thanks.